src/machine.rs

```diff
@@ -196,14 +203,30 @@ impl<'a, 'tcx> Machine<'a, 'tcx> {
 
     pub fn push_frame(&mut self,
                       def_id: DefId,
-                      ret_addr: Option<BasicBlock>) {
+                      mut args: Vec<TypedVal<'tcx>>,
+                      ret_val: Option<Address>,
+                      ret_block: Option<BasicBlock>){
         let def = Instance::mono(self.tcx, def_id).def;
         let mir = self.load_mir(def);
-        let frame = Frame::new(def_id, mir, ret_addr);
+
+
+        // Initialise locals
+        let size = mir.local_decls.len();
+        let mut tmp_locals = vec![None; size]; // Init to none
```

Comment:
I think the comment is obvious enough that it's not useful.

Reply:
True. Removed